grails-app/i18n/messages.properties

```diff
@@ -2734,5 +2735,61 @@ react.stockListManagement.emailSubject.label=STOCK LIST UPDATE
 react.stockListManagement.alert.emailSend.label=Email sent successfully
 react.stockListManagement.alert.noManagerAssociated.label=There is no manager associated with this stock list. Please add a manager and try again.
 react.stockListManagement.alert.noManagerSelected.label=Please add a manager as a recipient and resend.
+
+# Dashboard titles
+react.default.dashboard.numberData.inventoryByLotAndBin=Inventory Details by Lot and Bin
+react.default.dashboard.numberData.inProgressShipments=Your in Progress Shipments
+react.default.dashboard.numberData.inProgressPutaways=Your in Progress Putaways
+react.default.dashboard.numberData.receivingBin=Products in Receiving Bin
+react.default.dashboard.numberData.itemsInventoried=Items Inventoried this Month
+react.default.dashboard.numberData.defaultBin=Products in Default Bin
+react.default.dashboard.numberData.productWithNegativeInventory=Products with Negative Inventory
+react.default.dashboard.numberData.expiredProductsInStock=Expired products in Stock
+
+react.default.dashboard.indicatorData.expirationSummaryData=Expiration Summary
+react.default.dashboard.indicatorData.fillRate=Fill Rate Last 12 Months
+react.default.dashboard.indicatorData.fillRateSnapshot=Fill Rate Last Month
+react.default.dashboard.indicatorData.inventorySummaryData=Inventory Summary
+react.default.dashboard.indicatorData.sentStockMovements=Stock Movements Sent by Month
+react.default.dashboard.indicatorData.receivedStockData=Stock Movements Received by Month
+react.default.dashboard.indicatorData.outgoingStock=Outgoing Stock Movements in Progress
+react.default.dashboard.indicatorData.incomingStock=Incoming Stock Movements By Status
+react.default.dashboard.indicatorData.discrepancy=Items received with a discrepancy
+react.default.dashboard.indicatorData.productsInventoried=Percent of Products Inventoried
+react.default.dashboard.indicatorData.delayedShipments=Delayed Incoming Stock Movements
+react.default.dashboard.indicatorData.lossCausedByExpiry=Value of Stock Lost to Expiry
+react.default.dashboard.indicatorData.percentageAdHoc=Stock vs ad-hoc requests last month
+react.default.dashboard.indicatorData.stockOutLastMonth=Stock out last month
+
+# Dashboard labels lists
+react.default.dashboard.labels.timeline.today=today
```

Comment:
I think that `default` and `labels` in this case is redundant and the pattern is that `label` is the last part of a property.
So in your case it could be:
react.dashboard.numberData.XYZ.label
react.dashboard.indicatorData.XYZ.label
react.dashboard.timeline.XYZ.label, and so on.
We could also remove the `react` part from dashboard labels, but then you'd need to remember to include it in endpoint returning localization for react pages.

Reply:
Hello ! Yes, this is the part that I didn't understand. I have put react.default because the translation didn't work without. Where should I include this endpoint ? Maybe the problem come from here. @awalkowiak

Reply:
@julienreliefapps Quick explaining steps:
1.  As you can see here: https://github.com/openboxes/openboxes/blob/develop/src/js/actions/index.js#L123
there is the prefix that is added after the `react` beginning. 
2. In the MainRouter there is `this.props.fetchTranslations('', 'default');`, that's why it worked for you when you added the `default` to the property key.
3. In order to pull the `react.dashboard.xyz.abc.label`, you'd need to add in the main dashboard component `this.props.fetchTranslations('', 'dashboard');`.

Reply:
Ah ok I see ! I ll do that ! Thank you !

Reply:
Done !

Reply:
Done